keras/applications/efficientnet_v2.py

```diff
@@ -0,0 +1,1230 @@
+                      input_shape=None,
+                      pooling=None,
+                      classes=1000,
+                      classifier_activation='softmax',
+                      **kwargs):
+  return EfficientNetV2(
+      width_coefficient=1.0,
+      depth_coefficient=1.0,
+      default_size=384,
+      model_name="efficientnetv2-s",
+      include_top=include_top,
+      weights=weights,
+      input_tensor=input_tensor,
+      input_shape=input_shape,
+      pooling=pooling,
+      classes=classes,
+      classifier_activation=classifier_activation,
+      **kwargs)
+
+@keras_export('keras.applications.efficientnetv2.EfficientNetV2_M',
+              'keras.applications.EfficientNetV2_M')
+def EfficientNetV2_M(include_top=True,
+                      weights='imagenet',
+                      input_tensor=None,
+                      input_shape=None,
+                      pooling=None,
+                      classes=1000,
+                      classifier_activation='softmax',
+                      **kwargs):
+  return EfficientNetV2(
+      width_coefficient=1.0,
+      depth_coefficient=1.0,
+      default_size=480,
+      model_name="efficientnetv2-m",
+      include_top=include_top,
+      weights=weights,
+      input_tensor=input_tensor,
+      input_shape=input_shape,
+      pooling=pooling,
+      classes=classes,
+      classifier_activation=classifier_activation,
+      **kwargs)
+
+
+@keras_export('keras.applications.efficientnetv2.EfficientNetV2_L',
+              'keras.applications.EfficientNetV2_L')
+def EfficientNetV2_L(include_top=True,
+                      weights='imagenet',
+                      input_tensor=None,
+                      input_shape=None,
+                      pooling=None,
+                      classes=1000,
+                      classifier_activation='softmax',
+                      **kwargs):
+  return EfficientNetV2(
+      width_coefficient=1.0,
+      depth_coefficient=1.0,
+      default_size=480,
+      model_name="efficientnetv2-l",
+      include_top=include_top,
+      weights=weights,
+      input_tensor=input_tensor,
+      input_shape=input_shape,
+      pooling=pooling,
+      classes=classes,
+      classifier_activation=classifier_activation,
+      **kwargs)
+
+
+EfficientNetV2_B0.__doc__ = BASE_DOCSTRING.format(name='EfficientNetV2_B0')
+EfficientNetV2_B1.__doc__ = BASE_DOCSTRING.format(name='EfficientNetV2_B1')
+EfficientNetV2_B2.__doc__ = BASE_DOCSTRING.format(name='EfficientNetV2_B2')
+EfficientNetV2_B3.__doc__ = BASE_DOCSTRING.format(name='EfficientNetV2_B3')
+EfficientNetV2_S.__doc__ = BASE_DOCSTRING.format(name='EfficientNetV2_S')
+EfficientNetV2_M.__doc__ = BASE_DOCSTRING.format(name='EfficientNetV2_M')
+EfficientNetV2_L.__doc__ = BASE_DOCSTRING.format(name='EfficientNetV2_L')
+
+
+@keras_export('keras.applications.efficientnet_v2.preprocess_input')
+def preprocess_input(x, data_format=None):  # pylint: disable=unused-argument
```

Comment:
Our current policy is to *not* have a preprocessing function for new applications.

This function should just be pass-through, e.g.

```
def preprocess_input(x):
   return x
```

Meanwhile, the application itself should include a `Rescaling` layer as its first layer.

The goal being that applications should be able to process raw images, without the user having to care about what the correct preprocessing scheme to use is.

See e.g. 
https://github.com/keras-team/keras/blob/master/keras/applications/mobilenet_v3.py#L281-L282

Also include the following argument to make the Rescaling layer optional:

```
include_preprocessing: Boolean, whether to include the preprocessing
      layer (`Rescaling`) at the start of the network. Defaults to `True`.
```

https://github.com/keras-team/keras/blob/master/keras/applications/mobilenet_v3.py#L145